docs/HDRUK/3.0.0.form.json

```diff
@@ -24,7 +24,7 @@
                   "required": true,
                   "title": "Dataset Version",
                   "description": "Dataset metadata version.",
-                  "guidance": "N/A",
+                  "guidance": "Dataset metadata version should follow standard SEMVER naming conventions(https://semver.org). For example: 1.1.0 major.minor.patch.<br> Major: Significant/breaking changes.<br> Minor: New features and bug fixes.<br> Patch: Minor fixes without new features.\n",
```

Comment:
i would except more changes in this file and in the `.md` file when is see all the changes made to the annotations (which have propagated to `3.0.0.structure.json`

Reply:
actually forget that, these are changes to guidance rather than description

Reply:
Yep, this was a singular example, and not being familiar with the codebase i opted for a "single change and get everything working" approach. If this is all working fine i'll be doing a PR for changes to guidance/titles/descriptions etc etc :)